source/sessions/tests/README.rst

```diff
@@ -0,0 +1,71 @@
+====================
+Driver Session Tests
+====================
+
+.. contents::
+
+----
+
+Introduction
+============
+
+The YAML and JSON files in this directory are platform-independent tests that
+drivers can use to prove their conformance to the Driver Sessions Spec. They are
+designed with the intention of sharing most test-runner code with the
+Transactions spec tests.
+
+Several prose tests, which are not easily expressed in YAML, are also presented
+in the Driver Sessions Spec. Those tests will need to be manually implemented
+by each driver.
+
+Test Format
+===========
+
+The same as the Transactions spec tests format.
+
+Special Test Operations
+```````````````````````
+
+Certain operations that appear in the "operations" array do not correspond to
+API methods but instead represent special test operations. Such operations are
+defined on the "testRunner" object and are documented in the Transactions spec
```

Comment:
Just to confirm, is this the first use of "testRunner"? I seem to recall seeing it discussed before, but that might have been in the context of the FLE or Live Failover spec. I know Live Failover is no longer using this, but on the off chance FLE has some concept of it I'd suggest confirming with @kevinAlbs to make sure everything is consistent.

Reply:
The transaction spec already uses this concept for targetedFailPoint/assertSessionPinned/assertSessionUnpinned: https://github.com/mongodb/specifications/blob/master/source/transactions/tests/README.rst#special-test-operations